src/Microsoft.Framework.FileSystemGlobbing/Internal/Patterns/PatternBuilder.cs

```diff
@@ -8,12 +8,24 @@
 
 namespace Microsoft.Framework.FileSystemGlobbing.Internal.Patterns
 {
-    public static class PatternBuilder
+    public class PatternBuilder
     {
         private static readonly char[] _slashes = new[] { '/', '\\' };
         private static readonly char[] _star = new[] { '*' };
 
-        public static IPattern Build(string pattern)
+        public PatternBuilder()
+        {
+            ComparisonType = StringComparison.OrdinalIgnoreCase;
+        }
+
+        public PatternBuilder(StringComparison comparisonType)
```

Comment:
Doesn't look like these properties are used outside of `Build`. Why make the change to non-static when the parameter could be passed in?